thewalrus/fock_gradients.py

```diff
@@ -355,3 +357,89 @@ def grad_beamsplitter(T, theta, phi):  # pragma: no cover
                     )
 
     return grad_theta, grad_phi
+
+
+@jit(nopython=True)
+def mzgate(theta, phi, cutoff, dtype=np.complex128):  # pragma: no cover
+    r"""Calculates the Fock representation of the mzgate.
```

Comment:
```suggestion
    r"""Calculates the Fock representation of the Mach-Zehnder interferometer.
```

I feel like it makes more sense to refer to it as the Mach-Zehnder interferometer in documentation, because otherwise the definition is somewhat tautological?